src/GraphQL.Tests/CultureTestHelper.cs

```diff
@@ -14,6 +14,10 @@ public static class CultureTestHelper
             new CultureInfo(CultureInfo.CurrentCulture.Name)
         };
 
+        /// <summary>
+        /// Executes the specified delegate with a variety of cultures.
+        /// Be sure to mark the test class with <c>[Collection("StaticTests")]</c>
+        /// </summary>
```

Comment:
```suggestion
        /// Be sure to mark the test class with <c>[Collection("StaticTests")]</c>
        /// to avoid the interference of static variables.
        /// </summary>
```